_pages/interviews/core-values.md

```diff
@@ -80,6 +80,8 @@ This question could go in a few different directions depending on the candidates
         - What challenges did you face, if any, and how did you overcome them?
     - If no: What does agile software development mean to you?
 
+- [Talk to me about how you prioritize features](https://docs.google.com/document/d/1oYmx_93-mq2QrqICCo8SNk8hHmnPPonPA1kg0vhy540/edit#heading=h.z49dyqcuxle5)
```

Comment:
```suggestion
- [Talk to me about how you prioritize features](https://docs.google.com/document/d/1oYmx_93-mq2QrqICCo8SNk8hHmnPPonPA1kg0vhy540/edit#heading=h.z49dyqcuxle5).
```

Reply:
Thanks, @ctro!

I suggest:
1) Adding a period (suggested herein) and
2) Moving this question such that it's third instead of at the end. (Further context in doc.)

Reply:
Good suggestions.  I actually pushed a new commit right as I noticed your suggestion @fureigh  (Suggestion!? Cool!)